templates/internal/appName/http/handler.go.tpl

```diff
@@ -26,7 +39,10 @@ func Handler() http.Handler {
 	// Replace this with your routes
 	routes.Handle("/ping", handlers.Endpoint("ping", svc.ping))
 	routes.Handle("/pong", handlers.Endpoint("pong", svc.pong))
-
+{{- $extraRoutes := (stencil.GetModuleHook "http/extraRoutes") }}
```

Comment:
Can you add a comment above these denoting that they were added by module hooks? We don't have a good debugability story for module hooks w/o that right now.